libs/application/templates/parental-leave/src/fields/InReviewSteps/index.tsx

```diff
@@ -52,7 +55,11 @@ const InReviewSteps: FC<FieldBaseProps> = ({
   refetch,
   errors,
 }) => {
-  const { isRequestingRights } = useApplicationAnswers(application)
+  const {
+    isSelfEmployed,
+    isRequestingRights,
+    usePersonalAllowanceFromSpouse,
```

Comment:
Keep in mind that these are not boolean but `yes` / `no`. So it is not enough to check for `true` / `false` but you'll need to check `if (isSelfEmployed === YES) { ... }`